packages/pwa-kit-dev/bin/pwa-kit-dev.js

```diff
@@ -12,23 +12,39 @@ const WebSocket = require('ws')
 const program = require('commander')
 const validator = require('validator')
 const {execSync: _execSync} = require('child_process')
-const scriptUtils = require('../scripts/utils')
-const uploadBundle = require('../scripts/upload.js')
 const pkg = require('../package.json')
 const {getConfig} = require('pwa-kit-runtime/utils/ssr-config')
 
+const scriptUtils = (() => {
+    try {
+        return require('../dist/utils/script-utils')
+    } catch {
+        return require('../utils/script-utils')
+    }
```

Comment:
Not critical but could be as follows just a avoid abusing `try / catch`

```javascript
    try {
        return require('../dist/utils/script-utils')
    } catch (e) {
        console.log(e)
    }
    return require('../utils/script-utils')
```

Reply:
Oh nevermind... we don't want to surface the absence of `dist/utils/script-utils` to the user probably. Is there a meaningful error condition we would want to surface?